### DISCUSSION TITLE
Livewire not found

### DISCUSSION THREAD

**Discussion by Rakamishu** [LycheeOrg/Lychee#2318] [Q&A]
I have a fresh installation of Lychee 5 which is entirely broken because livewire.js cannot be found:
GET http://localhost/livewire/livewire.js?id=6b5eb707 net::ERR_ABORTED 404 (Not Found)

My installation is in a subfolder: http://localhost/Lychee/public 

I ran all commands in the console as per the documentation: 
composer install --no-dev
npm install
npm run build

**Comment by d7415** [comment]
Did you set your `APP_DIR`?

**Reply by Rakamishu** [reply]
```
# If using Lychee in a sub folder, specify the path after the tld here.
# For example for https://lychee.test/path/to/lychee
# Set APP_URL=https://lychee.test
# and APP_DIR=/path/to/lychee
# We (LycheeOrg) do not recommend the use of APP_DIR.
APP_DIR=/Lychee
```

I didn't but it doesn't seem to work.

**Reply by ildyria** [reply]
it should be `/Lychee/public` in your case.

**Reply by Rakamishu** [reply]
Yes, I tested a few paths, none of them worked.
`GET http://localhost/Lychee/public/livewire/livewire.js?id=6b5eb707 404 (Not Found)`

**Comment by Rakamishu** [comment]
So I went into this folder:
`D:\xampp\htdocs\Lychee\vendor\livewire\livewire\dist`
From which I copied the file livewire.js and pasted it into:
`D:\xampp\htdocs\Lychee\public\livewire`

Which fixed it. Is this how I am supposed to install it? This is absurd!

Edit: Now thumbnails don't load. Great. Anyway..

**Reply by d7415** [reply]
> Is this how I am supposed to install it?

No

> This is absurd!

Yes, that solution is absurd.
